src/Nethermind/Nethermind.Blockchain/BlockTree.cs

```diff
@@ -457,7 +457,10 @@ await using (cancellationToken.Register(() => _dbBatchProcessed.SetCanceled()))
             }
             catch (Exception ex)
             {
-                if (_logger.IsError) _logger.Error("Failed to load blocks from DB", ex);
+                if (!(ex is TaskCanceledException))
```

Comment:
use when

Reply:
I tried and gave up
let me see once again